homeassistant/components/bluesound/media_player.py

```diff
@@ -449,6 +447,22 @@ async def async_update_status(self):
             _LOGGER.info("Client connection error, marking %s as offline", self._name)
             raise
 
+    @property
+    def unique_id(self):
+        """Return an unique ID."""
+        return f"{format_mac(self._sync_status.get('@mac'))}-{self.port}"
+
+    @property
+    def device_info(self):
+        """Return a device description for device registry."""
+        return {
+            "identifiers": {(DOMAIN, self.unique_id)},
+            "name": self._sync_status.get("@name"),
```

Comment:
Is `@name` optional? Or can you refer it like this `self._sync_status[@name]`?

Reply:
I don't know, it is used as `get()` everywhere (there is no official documentation).